ui/app/components/job-page/parts/summary.js

```diff
@@ -3,7 +3,7 @@ import { action, computed } from '@ember/object';
 import { inject as service } from '@ember/service';
 import { classNames } from '@ember-decorators/component';
 import classic from 'ember-classic-decorator';
-
+import { camelize } from '@ember/string';
```

Comment:
Pretty minor but it looks like here and elsewhere a newline between he imports and the export got dropped.